ext/standard/basic_functions.stub.php

```diff
@@ -265,7 +265,7 @@ function long2ip(int $ip): string|false {}
 function getenv(?string $name = null, bool $local_only = false): string|array|false {}
 
 #ifdef HAVE_PUTENV
-function putenv(string $assignment): bool {}
+function putenv(string $env_var): bool {}
```

Comment:
I know that I suggested `$assignment`, but now, I'd like to propose `$env_var`, simply.

Reply:
I'm not a fan of `$env_var` here: `$env_var` sounds like the name of an environment variable to me, like what you would pass to `getenv()`. The `$assignment` here captures the fact that this is expects `"FOO=bar"`. The manual calls this `$setting`.

Reply:
OK, let's stick with `$assignment`